spec/jobs/ignored_users_summary_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'rails_helper'
+
+require_dependency 'jobs/scheduled/ignored_users_summary'
+
+describe Jobs::IgnoredUsersSummary do
+  before do
+    SiteSetting.ignore_user_enabled = true
+    SiteSetting.ignored_users_count_message_threshold = 1
+    SiteSetting.ignored_users_message_gap_days = 365
+  end
+
+  subject { Jobs::IgnoredUsersSummary.new.execute({}) }
+
+  context "when no ignored users" do
+    it "does nothing" do
+      subject
+      expect(Post.count).to eq(0)
```

Comment:
This should test that you're (not) creating a system message to the moderators group instead of "just" a post.